dev-db/mariadb/mariadb-10.6.11-r1.ebuild

```diff
@@ -0,0 +1,1321 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
```

Comment:
Can you verify if it's possible to move to EAPI=8, as you are already revbumping?

Reply:
Thanks, updated to EAPI 8